shipit-engine.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.rdoc"]
   s.test_files = Dir["test/**/*"]
 
-  s.add_dependency 'rails', '~> 4.2.2'
+  s.add_dependency 'rails', '~> 4.2.0'
```

Comment:
Yeah, definitely not this gem job to make sure people don't use vulnerable
versions :+1:

On Fri, Jul 17, 2015, 13:32 Jean Boussier <notifications@github.com> wrote:

> In shipit-engine.gemspec
> <https://github.com/Shopify/shipit-engine/pull/459#discussion_r34906289>:
>
> > @@ -16,7 +16,7 @@ Gem::Specification.new do |s|
> >    s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.rdoc"]
> >    s.test_files = Dir["test/**/*"]
> >
> > -  s.add_dependency 'rails', '~> 4.2.2'
> > +  s.add_dependency 'rails', '~> 4.2.0'
>
>  If it is worth noting, 4.2.1 is vulnerable to some security issues
>
> Yeah I guess we can merge this anyway.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Shopify/shipit-engine/pull/459/files#r34906289>.
>